src/test/kotlin/net/maizegenetics/phgv2/cli/LoadVCFTest.kt

```diff
@@ -1,30 +1,185 @@
 package net.maizegenetics.phgv2.cli
 
 import com.github.ajalt.clikt.testing.test
+import com.google.common.io.Files
+import net.maizegenetics.phgv2.utils.VariantLoadingUtilsTest
+import net.maizegenetics.phgv2.utils.bgzipAndIndexGVCFfile
+import org.junit.jupiter.api.AfterAll
+import org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.BeforeAll
+import org.junit.jupiter.api.assertThrows
 import org.junit.jupiter.api.extension.ExtendWith
+import java.io.File
 import kotlin.test.Test
 import kotlin.test.assertEquals
 
 @ExtendWith(TestExtension::class)
 class LoadVCFTest {
     companion object {
+        val tempDir = "${System.getProperty("user.home")}/temp/phgv2Tests/tempDir/"
```

Comment:
I think TestExtension does all this for us

Reply:
When we initially had the companion objects, TestExtension wasn't all setup.   Some files are still not pointed to it (e.g. the test/smallSeq/data folder shoudl be linked).  I'll work on cleaning up all the test references once I had the basics working for CreateRefVCF